src/modules/file-share/stop-file-share/stop-file-share.js

```diff
@@ -0,0 +1,52 @@
+(function() {
+    "use strict";
+
+    var app = angular.module("TendrlModule");
+
+    app.controller("StopFileShareController", StopFileShareController);
+
+    /*@ngInject*/
+    function StopFileShareController($rootScope, $scope, $state, selectedFileShare, fileShareStore) {
```

Comment:
We will need to put the efforts again to replace fileshare with volume so please change it accordingly.